script.js

```diff
@@ -0,0 +1,313 @@
+  document.querySelector('.wrapper-sub-head .work-subtext .canopy').textContent = worksArray[j].subheadText1;
+  document.querySelector('.wrapper-sub-head .work-subtext .description').textContent = worksArray[j].subheadText2;
+  document.querySelector('.wrapper-sub-head .work-subtext .description2').textContent = worksArray[j].subheadText3;
+  document.querySelector('.wrapper-img .work-image img').src = worksArray[j].image1;
+  document.querySelector('.mdl-fullscreen .work-image-large img').src = worksArray[j].image2;
+  document.querySelector('.wrapper-text .long-paragraph').textContent = worksArray[j].paragraph;
+  document.querySelector('.wrapper-text p .span1').textContent = worksArray[j].language1;
+  document.querySelector('.wrapper-text p .span2').textContent = worksArray[j].language2;
+  document.querySelector('.wrapper-text p .span3').textContent = worksArray[j].language3;
+  document.querySelector('.mdl-fullscreen .mdl-text p').textContent = worksArray[j].paragraph2;
+  document.querySelector('.mdl-fullscreen .mdl-text p .span4').textContent = worksArray[j].language1;
+  document.querySelector('.mdl-fullscreen .mdl-text p .span5').textContent = worksArray[j].language2;
+  document.querySelector('.mdl-fullscreen .mdl-text p .span6').textContent = worksArray[j].language3;
+  document.querySelector('.mdl-fullscreen .mdl-text p .span7').textContent = worksArray[j].language4;
+  document.querySelector('.mdl-fullscreen .mdl-text p .span8').textContent = worksArray[j].language5;
+  document.querySelector('.mdl-fullscreen .mdl-text p .span9').textContent = worksArray[j].language6;
+  document.querySelector('.wrapper .wrapper-btns .btnA').textContent = 'See Live';
+  document.querySelector('.wrapper .wrapper-btns .btnB').textContent = 'See Source';
+  document.querySelector('.mdl-fullscreen .mdl-text .btnA').textContent = 'See Live';
+  document.querySelector('.mdl-fullscreen .mdl-text .btnB').textContent = 'See Source';
+  divSpan1.addEventListener('click', () => {
+    popupDiv.remove();
+  });
+  seeLiveBtn.addEventListener('click', () => {
+    window.open(worksArray[j].liveBtn);
+  });
+  seeSourceBtn.addEventListener('click', () => {
+    window.open(worksArray[j].sourceBtn);
+  });
+}
+
+btn1.addEventListener('click', () => {
+  j = 0;
+  popup();
+});
+
+btn2.addEventListener('click', () => {
+  j = 1;
+  popup();
+});
+
+btn3.addEventListener('click', () => {
+  j = 2;
+  popup();
+});
+
+btn4.addEventListener('click', () => {
+  j = 3;
+  popup();
+});
+
+const form = document.querySelector('form');
+const emailAdrdress = document.getElementById('email');
+const emailPattern = /^[a-z0-9._-]+@[a-z0-9.-]+\.[a-z]{2,4}$/g;
+const msgError = document.querySelector('#message-error');
+const msgSuccess = document.querySelector('#message-success');
+
+form.addEventListener('submit', (e) => {
+  if (!emailAdrdress.value.match(emailPattern)) {
+    e.preventDefault();
+    msgError.classList.toggle('error');
```

Comment:
- [ ] If a user submits the contact form with the email field containing uppercase characters an error message should show up. But may be you have mistakenly toggling `error` css class instead of `hidden` class.  But there is no error message showing when the form is submitted with the email field containing uppercase characters.   Please make sure you fix this problem.